doc/source/ray-overview/learn-more.md

```diff
@@ -1,8 +1,25 @@
 # Learn More
 
-Here are some talks, papers, and press coverage involving Ray and its libraries.
+Here are some general rules of thumb, talks, papers, and press coverage involving Ray and its libraries.
 Please raise an issue if any of the below links are broken, or if you'd like to add your own talk!
 
+## Latency numbers every Ray programmer should know
+
+| Stat | Value |
+| --- | --- | 
+| Typical Ray cluster startup time | ~6 minutes |
+| Max size of Ray cluster | ~500 machines |
+| Typical base image size (compressed) | ~4GB | 
+| Typical base image size for GPU (compressed) | ~9GB | 
+| Max number of actors | ~10,000 |
```

Comment:
Same as above, let's consolidate into the Scalability Envelope?

Reply:
yeah, worth note following

> All distributed tests are run on 64 nodes with 64 cores/node. Maximum number of nodes is achieved by adding 4 core nodes.

Reply:
See comment above - would it make sense to roll the Scalability Envelope information into the documentation?

Related issue: [#22551](https://github.com/ray-project/ray/issues/22551)